explainability-service/src/main/java/org/kie/trustyai/service/payloads/data/upload/ModelInferRequestPayload.java

```diff
@@ -2,23 +2,15 @@
 
 import java.util.Arrays;
 
-public class ModelInferRequestPayload {
+public class ModelInferRequestPayload extends ModelInferBasePayload {
     private String id;
```

Comment:
Do we need this `id`? Could be inherited from a protected `ModelInferBasePayload`.